packages/zoltan/src/CMakeLists.txt

```diff
@@ -76,7 +76,7 @@ APPEND_SET(HEADERS
 
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/all)
 
-APPEND_SET(NOINSTALLHEADERS
+APPEND_SET(INSTALLHEADERS
```

Comment:
Required to install header files for needed includes in the Zoltan Callbacks in Zoltan2's AlgZoltan